### PR TITLE
Skunkworks - AtlasCLI copy and docs (Part 1)

### PR DESCRIPTION
**Pull request by blva**
## Proposed changes

Process: 
1. Each person took a set of commands to review
2. Each one wrote a set of suggestions for their commands
3. Each one reviews the docs by the other ones
4. Bianca writes the updates 

[[copy] atlas deployments setup by Corry](https://github.com/mongodb/mongodb-atlas-cli/pull/2399/commits/6c6e5d74a93816bf29c539e5bbc21a3495d21db9)
[6c6e5d7](https://github.com/mongodb/mongodb-atlas-cli/pull/2399/commits/6c6e5d74a93816bf29c539e5bbc21a3495d21db9)
[atlas config describe by Sarah](https://github.com/mongodb/mongodb-atlas-cli/pull/2399/commits/f52e90444562b3f41cdd37ae312f2bc4158582b7)
[f52e904](https://github.com/mongodb/mongodb-atlas-cli/pull/2399/commits/f52e90444562b3f41cdd37ae312f2bc4158582b7)
[flag usage by Corry](https://github.com/mongodb/mongodb-atlas-cli/pull/2399/commits/12bb915403075a1205466442c0330d043d5e76af)
[12bb915](https://github.com/mongodb/mongodb-atlas-cli/pull/2399/commits/12bb915403075a1205466442c0330d043d5e76af)
[atlas config set by Carrie](https://github.com/mongodb/mongodb-atlas-cli/pull/2399/commits/9d2be4104794925c920dc8ff62d30b175dc27c8a)
[9d2be41](https://github.com/mongodb/mongodb-atlas-cli/pull/2399/commits/9d2be4104794925c920dc8ff62d30b175dc27c8a)
[atlas accesslists create by Carrie](https://github.com/mongodb/mongodb-atlas-cli/pull/2399/commits/f890158b0cca568b740c3a92f0eadff946f5b40d)
[f890158](https://github.com/mongodb/mongodb-atlas-cli/pull/2399/commits/f890158b0cca568b740c3a92f0eadff946f5b40d)

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by fmenezes — APPROVED** [review]
make sure to get a review from docs

**Review by wtrocki — APPROVED** [review]

File: internal/usage/usage.go [hunk 1]
```diff
@@ -7,7 +7,7 @@
 //      http://www.apache.org/licenses/LICENSE-2.0
 //
 // Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
+// distributed under the License is distributed on an "AS IS"BASIS,
```
Comment (sarahsimpers) [inline internal/usage/usage.go:10]: Did we mean to change this?

File: internal/usage/usage.go [hunk 2]
```diff
@@ -440,16 +440,16 @@ dbName and collection are required only for built-in roles.`
...
+	CompletedAfter                            = "Date filter for when the backup snapshots were completed. You must use the YYYY-MM-DD format."
 	Tag                                       = "List that contains key-value pairs between 1 to 255 characters in length for tagging and categorizing the cluster."
 	DeploymentTag                             = "List that contains key-value pairs between 1 to 255 characters in length for tagging and categorizing the deployment."
 	ServerlessTag                             = "List that contains key-value pairs between 1 to 255 characters in length for tagging and categorizing the serverless instance."
-	UpdateWarning                             = " Passing this flag replaces preexisting data."
-	EnableWatch                               = "Watch the command until it completes its execution or the watch times out. To set the time that the watch times out, use the --watchTimeout option."
-	EnableWatchDefault                        = "Watch the command until it completes its execution or the watch times out."
+	UpdateWarning                             = "Flag that indicates whether passing this flag replaces preexisting data."
```
Comment (sarahsimpers) [inline internal/usage/usage.go:450]: I haven't seen the original copy suggestion, but I think this makes more sense
```suggestion
	UpdateWarning                             = "Flag that indicates whether to replace preexisting data."
```

File: internal/usage/usage.go [hunk 3]
```diff
@@ -403,33 +403,33 @@ dbName and collection are required only for built-in roles.`
...
+	EnableJavascript                          = "Flag that enables the execution of operations that perform server-side executions of JavaScript."
 	OplogMinRetentionHours                    = "Minimum retention window for cluster's oplog expressed in hours."
 	OplogSizeMB                               = "Storage limit of cluster's oplog expressed in megabytes."
 	SampleRefreshIntervalBIConnector          = "Interval in seconds at which the mongosqld process re-samples data to create its relational schema."
 	SampleSizeBIConnector                     = "Number of documents per database to sample when gathering schema information."
 	ExporterClusterName                       = "One or more comma separated cluster names to import"
-	OperatorIncludeSecrets                    = "Generate kubernetes secrets with data for projects, users, deployments entities" //nolint:gosec //This is just a message, not a credential
+	OperatorIncludeSecrets                    = "Flag that generates kubernetes secrets with data for projects, users, deployments entities."
```
Comment (sarahsimpers) [inline internal/usage/usage.go:417]: I believe the comment that was removed here is required to pass linting due to the use of the word "secrets"? But not sure

**Review by sarahsimpers — COMMENTED** [review]
Question for @corryroot who did the copy review for usage.go... how do we want to determine which descriptions we start with "Flag that...." since they are all flags when used in a CLI (as opposed to the API)? Are we using it here only when the flag itself does the enabling/disabling (so you specify "--flag" to enable something, not "--flag true"?) Just making sure I understand. If it's the latter, I'm not sure whether all of the replacements we made for ones that said "if true" or "if false" make sense, but will defer to you on this. Thanks!

File: internal/usage/usage.go (hunk 1 above)
Comment (blva) [reply internal/usage/usage.go:10]: no, thanks!

File: internal/usage/usage.go (hunk 2 above)
Comment (blva) [reply internal/usage/usage.go]: updated! keeping this for history

File: internal/usage/usage.go [hunk 4]
```diff
@@ -222,35 +222,35 @@ dbName and collection are required only for built-in roles.`
...
-	Normalization                             = "Flag that, if set to true, collation checks if text requires normalization and performs normalization to compare text."
-	Backwards                                 = "Flag that, if set to true, strings with diacritics sort from the back to the front of the string."
+	Alternate                                 = "Flag that determines whether collation should consider whitespace and punctuation as base characters during comparisons."
+	MaxVariable                               = "Flag that determines which characters to ignore. This flag applies only if you set indexConfigs.collation.alternate to shifted."
+	NumericOrdering                           = "Flag that indicates that collation compares numeric strings as numbers. If you set to false, collation compares numeric strings as strings."
+	Normalization                             = "Flag that indicates whether collation checks if text requires normalization and performs normalization to compare text."
+	Backwards                                 = "Flag that indicates whether strings with diacritics sort from the back to the front of the string."
```
Comment (corryroot) [inline internal/usage/usage.go:247]: **Question:**

@dianchenghu and @blva, are all booleans passed as simple flags, or do users actually have to specify "true" and "false"? See @sarahsimpers comment. If so, can we have a consistent behavior.

File: internal/usage/usage.go [hunk 5]
```diff
@@ -300,24 +300,24 @@ dbName and collection are required only for built-in roles.`
 	Network                                   = "Unique identifier of the Network Peering connection in the Atlas project."
 	APIRegion                                 = "Code that indicates which regional URL MongoDB uses to access the third-party API. Valid values are US and EU."
 	DatadogAPIRegion                          = "Code that indicates which regional URL MongoDB uses to access the Datadog API. Valid values are US, EU, US3, and US5."
-	SkipMongosh                               = "Indicates whether to skip accessing your deployment with MongoDB Shell."
-	SkipSampleData                            = "Indicates whether to skip loading sample data into your MongoDB cluster."
-	SkipSampleDataDeployment                  = "Indicates whether to skip loading sample data into your MongoDB deployment."
+	SkipMongosh                               = "Flag that indicates whether to skip accessing your deployment with MongoDB Shell."
+	SkipSampleData                            = "Flag that ndicates whether to skip loading sample data into your MongoDB cluster."
```
Comment (corryroot) [inline internal/usage/usage.go:304]: **Typo:**

```suggestion
	SkipSampleData                            = "Flag that indicates whether to skip loading sample data into your MongoDB cluster."
```

File: internal/usage/usage.go [hunk 6]
```diff
@@ -403,33 +403,33 @@ dbName and collection are required only for built-in roles.`
...
-	OperatorAtlasGov                          = "Flag to configure Atlas for Government as target of the operator."
+	OperatorImport                            = "Flag that indicates whether to import existing Atlas resources into the cluster for the operator to manage."
+	OperatorAtlasGov                          = "Flag that indicates whether to configure Atlas for Government as a target of the operator."
 	KubernetesClusterConfig                   = "Path to the kubeconfig file to use for CLI requests."
-	KubernetesClusterContext                  = "The name of the kubeconfig context to use."
+	KubernetesClusterContext                  = "Name of the kubeconfig context to use."
 	ExportID                                  = "Unique string that identifies the AWS S3 bucket to which you export your snapshots."
-	RequiredRole                              = "To use this command, you must authenticate with a user account or an API key that has the %s role."
+	RequiredRole                              = "Role required to use this command. You must authenticate with a user account or an API key that has the %s role."
```
Comment (corryroot) [inline internal/usage/usage.go:429]: @dianchenghu and @blva, sorry, since this isn't a typical flag description, I think we should change this back.

**Review by corryroot — COMMENTED** [review]
@blva and @dianchenghu, I have a follow up question to @sarahsimpers's comment. I noticed a very small typo and want to undo one of the changes.

File: internal/usage/usage.go (hunk 4 above)
Comment (blva) [reply internal/usage/usage.go:247]: any flag marked with `BoolVar` will be just passed as flags indeed, nobody needs to specify true, but some will have default to "true" and some will have default to "false"

**Review by sarahsimpers — DISMISSED** [review]

**Comment by sarahsimpers** [comment]
LGTM

File: internal/usage/usage.go (hunk 4 above)
Comment (dianchenghu) [reply internal/usage/usage.go:247]: @corryroot  We set default values for each bool flag in code if they're not specified.
For each of the above under index create, the default is false.
So "indicates that" would probably be better for them.

**Review by corryroot — COMMENTED** [review]
@blva, I spotted just one more that we should change back. Otherwise, this LGTM!

File: internal/usage/usage.go [hunk 7]
```diff
@@ -440,16 +440,16 @@ dbName and collection are required only for built-in roles.`
...
+	CompletedAfter                            = "Date filter for when the backup snapshots were completed. You must use the YYYY-MM-DD format."
 	Tag                                       = "List that contains key-value pairs between 1 to 255 characters in length for tagging and categorizing the cluster."
 	DeploymentTag                             = "List that contains key-value pairs between 1 to 255 characters in length for tagging and categorizing the deployment."
 	ServerlessTag                             = "List that contains key-value pairs between 1 to 255 characters in length for tagging and categorizing the serverless instance."
-	UpdateWarning                             = " Passing this flag replaces preexisting data."
-	EnableWatch                               = "Watch the command until it completes its execution or the watch times out. To set the time that the watch times out, use the --watchTimeout option."
-	EnableWatchDefault                        = "Watch the command until it completes its execution or the watch times out."
+	UpdateWarning                             = "Flag that indicates whether to replace preexisting data."
```
[the remark this answers is not in the document]
Comment (blva) [reply internal/usage/usage.go:450]: are you refering to only UpdateWarning?

[the remark this answers is not in the document]
Comment (corryroot) [reply internal/usage/usage.go:450]: Yes, back to:
```
" Passing this flag replaces preexisting data."
```

**Review by corryroot — APPROVED** [review]
LGTM!

**Pull request merged**